src/zpotrf_L.jdf

```diff
@@ -524,6 +532,24 @@ END
 
 extern "C" %{
 
+/* Compute the time estimates based on device capabilities and flops for the task */
+static int64_t zpotrf_time_estimate(const parsec_task_t *task, parsec_device_module_t *dev) {
+    int mb = ((parsec_zpotrf_L_taskpool_t*)task->taskpool)->_g_descA->mb;
+    return (int64_t)FLOPS_ZPOTRF(mb) / dev->gflops_fp64;
+}
+static int64_t ztrsm_time_estimate(const parsec_task_t *task, parsec_device_module_t *dev) {
+    int mb = ((parsec_zpotrf_L_taskpool_t*)task->taskpool)->_g_descA->mb;
+    return (int64_t)FLOPS_ZTRSM(PlasmaRight, mb, mb) / dev->gflops_fp64;
+}
+static int64_t zherk_time_estimate(const parsec_task_t *task, parsec_device_module_t *dev) {
+    int mb = ((parsec_zpotrf_L_taskpool_t*)task->taskpool)->_g_descA->mb;
+    return (int64_t)FLOPS_ZHERK(mb, mb) / dev->gflops_fp64;
```

Comment:
(Not for this PR) Is there any reason it returns int64_t instead of double? (Although I think it's OK for normal use where mb is bigger enough.)